app/parsers/bulkrax/csv_parser.rb

```diff
@@ -325,6 +329,7 @@ def write_files
       require 'open-uri'
       folder_count = 0
       sorted_entries = sort_entries(importerexporter.entries.uniq(&:identifier))
+                       .select { |e| valid_entry_types.include?(e.type) }
```

Comment:
now that we're exporting all work types as "Bulkrax::CsvEntry", we don't need to reference the other custom work entry types that may have been previously exported. the valid work itself will still be exported because of the `#create_new_entries` method. but it will be only as a "Bulkrax::CsvEntry".